### DISCUSSION TITLE
【作业提交】zkiap session 1

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [zkp-co-learning/zkp-co-learn#27] [Show and tell]
把你的作业 repo 提交到评论区吧~ 

1、office hour 的时候可以直接打开某条作业点评
2、便于从 participates 区域统计大家的作业完成情况

**Comment by xiegudong45** [comment]
https://github.com/xiegudong45/MIT-IAP/tree/main/session1

**Comment by abbylow** [comment]
https://github.com/abbylow/zkiap/blob/main/session-1-exercise.md
